packages/core/src/fhir-to-cda/cda-templates/commons.ts

```diff
@@ -190,3 +205,94 @@ export function formatDateToCDATimeStamp(dateString: string | undefined): string
   const cdaTimeStamp = `${datePart}${timePart}.${fractionalSeconds}`;
   return cdaTimeStamp;
 }
+
+function mapAddressUse(use: string | undefined) {
+  if (!use) return undefined;
+  // From PostalAddressUse of the CDA R2 IG
+  switch (use.toLowerCase()) {
+    case "bad address":
+      return "BAD";
+    case "confidential":
+      return "CONF";
+    case "direct":
+      return "DIR";
+    case "home" || "home address":
+      return "H";
+    case "primary home":
+      return "HP";
+    case "vacation home":
+      return "HV";
+    case "physical visit address":
+      return "PHYS";
+    case "postal address":
+      return "PST";
+    case "public":
+      return "PUB";
+    case "temporary":
+      return "TMP";
+    // from example CDAs
+    case "work":
+      return "WP";
+  }
+  return use;
+}
+
+function mapTelecomUse(use: string | undefined) {
+  if (!use) return undefined;
+  // From Telecom Use of the CDA R2 IG
+  switch (use.toLowerCase()) {
+    case "answering service":
+      return "AS";
+    case "emergency contact":
+      return "EC";
+    case "home" || "primary home":
+      return "HP";
+    case "vacation home":
+      return "HV";
+    case "mobile contact":
+      return "MC";
+    case "pager":
+      return "PG";
+    case "work" || "work place":
+      return "WP";
+  }
+  return use;
+}
+function cleanUpCoding(primaryCodingRaw: import("@medplum/fhirtypes").Coding | undefined) {
```

Comment:
this in line import is weird? why not just do it at the top of the file?

Reply:
Woops, missed this!